### DISCUSSION TITLE
The thing doesn't want to remain open

### DISCUSSION THREAD

**Discussion by ehjr5u** [Purfview/whisper-standalone-win#313] [Q&A]
I just unziped it and when I open the .exe, it shows some things and immediately closes after.

**Comment by Purfview** [comment]
You don't "open" CLI tools, you pass a command to it.

If you don't know what CLI is then watch a guide linked on the main page.

**Reply by ehjr5u** [reply]
Man just cut to the chase just tell me how can i set it up?

**Reply by Purfview** [reply]
>Man just cut to the chase just tell me how can i set it up?

There is no such thing as "set it up", it's like using a fridge, open a door take that yogurt - PROFIT.

**Reply by ehjr5u** [reply]
To be more specific, i opened faster-whisper-xxl.exe and that gave me the closure. 

you gotta put instructions on your main page, PAL, and I'll just revert to using the traditional method if you're not solving my issue

**Reply by Purfview** [reply]
> To be more specific, i opened faster-whisper-xxl.exe and that gave me the closure.

I understood what you wrote from your first post. Read the answer.

> you gotta put instructions on your main page, PAL, and I'll just revert to using the traditional method if you're not solving my issue

Instructions are on the main page, PAL!
You can revert to the challenged ways, no one is forcing you to use the traditional software.
